.github/pull_request_template.md

```diff
@@ -1,16 +1,16 @@
-<!-- Fill in the blank: -->
-
-This PR ______, so that ____.
-
-### Checklist
-
-<!-- Additionally, updates should conform closely as possible to the [Definition of Done](https://github.com/ipfs/devgrants/tree/master/open-grants/ipfs-rust#definition-of-done) defined in the devgrant proposal. This includes: -->
-
-- [ ] **New** code is “linted” i.e. code formatting via rustfmt and language idioms via clippy
-- [ ] There are no extraneous changes like formatting, line reordering, etc. Keep the patch sizes small!
-- [ ] There are functional and/or unit tests written, and they are passing
-- [ ] There is suitable documentation. In our case, this means:
-    - [ ] Each command has a usage example and API specification
-    - [ ] Top-level commands with subcommands display usage instructions
-    - [ ] Rustdoc tests are passing on all code-level comments
-    - [ ] Differences between Rust’s IPFS implementation and the Go or JS implementations are explained
+<!-- Fill in the blank: -->

```

Comment:
This is confusing. I rebased against master and it wants to delete the entire file and add the entire file again?

Reply:
Yeah this looks strange. Lets have a look..

Reply:
This version has windows CRLF, not visible in github ui.

Reply:
Other files don't have CRLF so if isn't required for some github specific reason I wouldn't keep them in to avoid big diffs in the future.

Reply:
Fixed